code/__HELPERS/mobs.dm

```diff
@@ -186,43 +186,49 @@ GLOBAL_LIST_EMPTY(species_list)
 
 /proc/do_mob(mob/user , mob/target, time = 30, uninterruptible = 0, progress = 1, datum/callback/extra_checks = null)
 	if(!user || !target)
-		return 0
+		return FALSE
 	var/user_loc = user.loc
 
 	var/drifting = FALSE
 	if(SSmove_manager.processing_on(user, SSspacedrift))
-		drifting = 1
+		drifting = TRUE
+
+	LAZYADD(user.do_afters, target)
+	LAZYADD(target.targeted_by, user)
```

Comment:
I hate that this is added specifically just for brute/burn kits, especially since this add event more references. I'm sure there are other things in the game which don't let you use multiple at once.

Reply:
It was an oversight that it didn't have these in the first place.

Reply:
For reference I added the `do_afters` list to mobs in the blind people can examine stuff PR, but I forgot to add it to `do_mob` as well. 

If there are other places like this they should be converted to use the `do_after` list instead of whatever they're doing.